types/dwt/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for dwt 12.3
+// Type definitions for dwt 13.1
```

Comment:
Could you make v12 folder and copy old files to that dir?
please see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version